src/context/categories/constants.js

```diff
@@ -14,6 +14,7 @@ export const getAllCategoriesQuery = gql`
           large
         }
         available
+        translations_key
```

Comment:
use camelCase instead of snake_case

Reply:
It's general approach in the project. And this field wrote in the same way in all the other components.

Reply:
Changed